### PR TITLE
fix(jest): include tsx files in coverage

### PR DESCRIPTION
**Pull request by marbemac**

### CODE REVIEW

File: jest-preset.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
   testMatch: ['<rootDir>/src/**/__tests__/*.(ts|js)?(x)'],
   moduleFileExtensions: ['ts', 'tsx', 'js'],
   coveragePathIgnorePatterns: ['/node_modules/', '/__tests__/', 'index.ts', 'types.ts'],
-  collectCoverageFrom: ['src/**/*.{js,ts}'],
+  collectCoverageFrom: ['src/**/*.{ts,tsx}'],
```
Comment (P0lip) [inline jest-preset.js]: I'd keep `js` for backward compatibility. WDYT?

Comment (rainum) [reply jest-preset.js]: we can assume that`.js` files is some kind of imported legacy code or 3rd party library which we do not want to test. All our code should be in a `.ts|.tsx` files IMO.

Comment (P0lip) [reply jest-preset.js]: Fair enough. Will merge it then.

**Pull request merged**